### PR TITLE
4playerfix

### PR DESCRIPTION
**Pull request by mrdudz**
the promised fix for the hitjoy driver :) includes some tweaks to the joy-test program

oh, and the FIXME is basically a reminder - the "problem" existed before and it doesnt really matter either.

### CODE REVIEW

File: testcode/lib/joy-test.c [hunk 1]
```diff
@@ -5,14 +5,29 @@
...
 #ifdef JOYSTICK_DRIVER
 
 /* A statically linked driver was named on the compiler's command line.
 ** Make sure that it is used instead of a dynamic one.
 */
 #  undef DYN_DRV
 #  define DYN_DRV       0
+
+/*
+ * link existing drivers like this:
```
Comment (oliverschmidt) [inline testcode/lib/joy-test.c]: Could you please adjust the comment style on this and the following lines to match the exsisting style on line 11?

File: testcode/lib/joy-test.c [hunk 2]
```diff
@@ -56,23 +74,31 @@ int main (void)
...
                      JOY_DOWN(j)?  " down " : " ---- ",
                      JOY_LEFT(j)?  " left " : " ---- ",
                      JOY_RIGHT(j)? "right " : " ---- ",
-                     JOY_BTN_1(j)? "button" : " ---- ");
+                     JOY_BTN_1(j)? "button" : " ---- ", j);
 #endif
         }
+
+        /* show pressed key, so we can verify keyboard is working */
+        kb = kbhit();
```
Comment (oliverschmidt) [inline testcode/lib/joy-test.c]: Please insert a space in front of the parameter parenthesis here and on line 97, 99, 100 and 101.

**Comment by mrdudz** [comment]
like that? :)

**Comment by oliverschmidt** [comment]
Yep :-)

**Pull request merged**